drivers/sensor/fdc2x1x/Kconfig

```diff
@@ -7,7 +7,7 @@ menuconfig FDC2X1X
 	bool "FDC2X1X Capacitance-to-Digital Converter"
 	default y
 	depends on DT_HAS_TI_FDC2X1X_ENABLED
-	depends on NEWLIB_LIBC
+	depends on FULL_LIBC_SUPPORTED
```

Comment:
I don't think these are compatible (replacing newlib with full_libc will bloat the image size of projects that use this sensor.

Reply:
Based on my reading of `lib/libc/Kconfig`...

```
config NEWLIB_LIBC_SUPPORTED
	bool
	default y
	depends on "$(TOOLCHAIN_HAS_NEWLIB)" = "y"
	select FULL_LIBC_SUPPORTED
```
...it seems like newlib satisfies the requirements for `FULL_LIBC_SUPPORTED`? I could also just make it `NEWLIB_LIBC || EXTERNAL_LIBC`.